WordPress/AbstractClassRestrictionsSniff.php

```diff
@@ -123,6 +123,15 @@ public function is_targetted_token( $stackPtr ) {
 
 		if ( \in_array( $token['code'], array( \T_NEW, \T_EXTENDS, \T_IMPLEMENTS ), true ) ) {
 			if ( \T_NEW === $token['code'] ) {
+				$nextNonEmpty = $this->phpcsFile->findNext( Tokens::$emptyTokens, ( $stackPtr + 1 ), null, true );
+
+				if ( false === $nextNonEmpty
+					|| \in_array( $this->tokens[ $nextNonEmpty ]['code'], array( \T_READONLY, \T_ANON_CLASS, \T_ATTRIBUTE ), true )
+				) {
+					// Live coding or anonymous class (bow out for anonymous classes as they don't have a name).
```

Comment:
```suggestion
					// Live coding or anonymous class. Bow out.
```

(it's not as if you 're not bailing out for live coding, now is it ?)